### ISSUE TITLE
Word counter displays incorrect value

### ISSUE THREAD

**Issue by Yunoxa** [saicaca/hexo-theme-vivia#55]
The word counter value shown on posts displays the symbol/character count instead of word count.

![image](https://github.com/saicaca/hexo-theme-vivia/assets/58912199/edd9f777-5d7d-462a-a68c-959bc1ea8c44)

**Comment by saicaca** [comment]
Yes, I noticed it. It was working fine before, and now I can't make it work again even if I roll back everything. I might need some more time to figure out the reason.

**Comment by Yunoxa** [comment]
Yeah, it seems a rather strange issue, nothing major though at least.
If I figure it out I'll let you know 👍

**Comment by JoeyC-Dev** [comment]
@saicaca Modify back to `hexo-wordcount` will fix the issue.
![image](https://github.com/saicaca/hexo-theme-vivia/assets/142381267/1d8b196c-c32e-4e9d-80c5-a6ffc13b50a3)
![image](https://github.com/saicaca/hexo-theme-vivia/assets/142381267/2f7ee431-76bf-4c92-8fd7-6a91369d4b4e)

![image](https://github.com/saicaca/hexo-theme-vivia/assets/142381267/2bf2b075-2091-43fc-8db8-0ece2355c328)
